core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionSupervisor.scala

```diff
@@ -26,56 +26,95 @@ import scala.util.control.NonFatal
 object SubmissionSupervisor {
   sealed trait SubmissionSupervisorMessage
 
-  case class SubmissionStarted(workspaceName: WorkspaceName, submissionId: UUID, credential: Credential)
+  case class SubmissionStarted(workspaceName: WorkspaceName, submissionId: UUID)
```

Comment:
It seemed stupid to pass in this credential every time a submission started, so I made it a constructor param instead.
@dvoet -- the credential itself doesn't expire, right?

Reply:
I think that is correct, probably bears some testing though.